agent/chain.py

```diff
@@ -64,52 +29,35 @@ def load_chains():
     """Logic for loading the chain you want to use should go here."""
     llm = ChatOpenAI(model_name = "gpt-4", temperature=1.2)
 
-    # chatGPT prompt formatting
-    objective_system_thought = SystemMessagePromptTemplate(prompt=OBJECTIVE_SYSTEM_THOUGHT)
-    objective_system_response = SystemMessagePromptTemplate(prompt=OBJECTIVE_SYSTEM_RESPONSE)
-
-    objective_human_thought = HumanMessagePromptTemplate(prompt=OBJECTIVE_HUMAN_THOUGHT)
-    objective_human_response = HumanMessagePromptTemplate(prompt=OBJECTIVE_HUMAN_RESPONSE)
-
-    objective_thought_chat_prompt = ChatPromptTemplate.from_messages([objective_system_thought, objective_human_thought])
-    objective_response_chat_prompt = ChatPromptTemplate.from_messages([objective_system_response, objective_human_response])
-
-    # define chains
-    objective_thought_chain = LLMChain(
-        llm=llm,
-        prompt=objective_thought_chat_prompt,
+    # define chain
+    bloom_chain = BloomChain(
+        llm=llm, 
         verbose=True
     )
 
-    objective_response_chain = LLMChain(
-        llm=llm,
-        prompt=objective_response_chat_prompt,
-        verbose=True
-    )
-
-    return ( 
-        objective_thought_chain, 
-        objective_response_chain,
-    )
+    return bloom_chain
 
 
 async def chat(**kwargs):
```

Comment:
See below comment on changes to `chat`. TL;DR is that we can make this another method within the `BloomChain` class that automatically does the thought response loop instead of requiring two separate calls.